extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -142,7 +142,7 @@ export class BaseCarousel extends AMP.BaseElement {
   }
 
   /** @override */
-  documentInactiveCallback() {
+  unlayoutCallback() {
```

Comment:
With this in mind, it looks like carousel should also have `pauseCallback` somewhere. I'm not sure if this can be deferred to the next PR.